osutil/mountinfo_linux.go

```diff
@@ -91,22 +91,17 @@ func (mi *MountInfoEntry) String() string {
 		flattenMap(mi.SuperOptions))
 }
 
-var mountInfoMustMockInTests = true
+var openMountInfoFile = func() (io.ReadCloser, error) {
+	CheckScopeIsNotTestExecution("/proc/self/mountinfo must be mocked in tests")
```

Comment:
It'd be great to have a unit test which executes this code path, i.e. calls openMountInfoFile without mocking it in the tests and expects a panic, something like:

```
func (s *mountinfoSuite) TestLoadMountInfoComplainWhenNotMockedInTest(c *C) {
	c.Assert(func() { osutil.LoadMountInfo() },
		PanicMatches, "/proc/self/mountinfo must be mocked in tests")
}
```